### ISSUE TITLE
Smasher instance only has 2TB available to it.

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#1733]
### Context

I was wondering how the smasher was doing space-wise:
```
$ df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            250G     0  250G   0% /dev
tmpfs            50G   25M   50G   1% /run
/dev/nvme0n1p1  2.0T  201G  1.8T  11% /
tmpfs           250G     0  250G   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           250G     0  250G   0% /sys/fs/cgroup
tmpfs            50G     0   50G   0% /run/user/1000
```

### Problem or idea

Turns out that it's partition is only 2TB instead of the 6 we wanted and have a volume for:
```
$ lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
nvme0n1     259:0    0  5.9T  0 disk 
└─nvme0n1p1 259:1    0    2T  0 part /
```

We should use the full 6TB that we have for that instance.

I tried to follow AWS' guide to doing this: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html

However when I ran the command to do so it gave an error:
```
$ sudo growpart /dev/nvme0n1 1
WARNING: MBR/dos partitioned disk is larger than 2TB. Additional space will go unused.
NOCHANGE: partition 1 could only be grown by 1 [fudge=2048]
```

According to https://unix.stackexchange.com/questions/429507/cannot-extend-partition-beyond-2tb-on-aws-ubuntu

> Your drive is formatted as MBR.

> For drives larger than 2TB, they need to be partitioned as GPT as MBR is limited to 2TB regardless of the OS.

### Solution or next step

The internet has guides to do this: https://www.cyberciti.biz/tips/fdisk-unable-to-create-partition-greater-2tb.html

We should make `infrastructure/nomad-configuration/client-smasher-instance-user-data.tpl.sh` do this when the smasher starts up.

**Comment by davidsmejia** [comment]
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/raid-config.html

```sudo mdadm --create --verbose /dev/md0 --level=0 --name=MY_RAID --raid-devices=number_of_volumes device_name1 device_name2```

**Comment by kurtwheeler** [comment]
This was closed by #2552!
